Changelog.md

```diff
@@ -1,5 +1,13 @@
 ## Gazebo Fuel Tools 7.x
 
+### Gazebo Fuel Tools 7.2.1 (2023-03-22)
+
+1. Forward port 4.8.2
```

Comment:
I think we should include the changes from the forward ports instead of just mentioning the PR that did the porting. I see a lot of changes in https://github.com/gazebosim/gz-fuel-tools/compare/ignition-fuel-tools7_7.2.0...ign-fuel-tools7. It would be difficult to know at a glance which of those changes made it into `7.x`.

Reply:
Is the policy that we should include all changes in previous versions? So, when I forward port this to version 8, I would include all changes in 4, 7, and 8? In the end I would copy and past the changelog lines from each previous version. I put in this line because it seemed more efficient to refer the reader to the 4.8.2 changelog lines.

Reply:
> Is the policy that we should include all changes in previous versions? 

It does seem tedious, but we do have a [tool](https://github.com/gazebo-tooling/release-tools/blob/master/source-repo-scripts/source_changelog.bash) to automate this. IMO, as a reader of the changelog, jumping to another point in the changelog is not ideal. In this case, you were able to point to a specific version (4.8.2), which is helpful. If we're always able to point to a specific version, then that might be okay, but I'm not sure if that's the case.

Reply:
I've updated the changelog in [a2b1924](https://github.com/gazebosim/gz-fuel-tools/pull/339/commits/a2b1924c3809cf94b0f0f734a49723bff58dcb48)